spdm.rflx

```diff
@@ -254,29 +264,47 @@ package SPDM is
        PSK_Resp_With_Context    => 2)
    with Size => 2;
 
-   type Capabilities_Response is
+   type Capabilities_Response_1_0 is
```

Comment:
The version 1.1 messages should be a variant of the respective 1.0 messages. This is not easily possible right now as we don't have the version information available in the inner message. The parameterized messages implemented in Componolit/RecordFlux#609 are a potential solution (similar to how we want to handle TCP checksums). Please add an `ISSUE: Componolit/RecordFlux#609` and a brief description so we don't forget to refactor it later to *each* message that could be simplified.

Reply:
Fixed in https://github.com/Componolit/RecordFlux-specifications/pull/75/commits/cfa18103274b8fdc2a2f5ff1bb0c73e0e33e0645.